### PR TITLE
Add request parameter endpoints to supply_information for current_total_supply and est_circulating_supply.

### PR DESCRIPTION
**Pull request by derekpierre**
- [x] NU values are no longer rounded to 2 decimals
- [x] Add request parameters:
   URLs will be:
       - https://status.nucypher.network/supply_information (returns json supply information)
       - https://https://status.nucypher.network/supply_information?q=current_total_supply (simply returns a value)
       - https://https://status.nucypher.network/supply_information?q=est_circulating_supply (simply returns a value)

### CODE REVIEW

**Review by mswilkison — APPROVED** [review]

File: monitor/dashboard.py [hunk 1]
```diff
@@ -97,13 +96,38 @@ def add_supply_endpoint(self, flask_server: Flask):
...
+                    mimetype='application/json'
+                )
+            else:
+                # specific request query provided
+                if parameter == 'current_total_supply':
+                    current_total_supply = calculate_current_total_supply(economics)
+                    response = flask_server.response_class(
+                        response=str(current_total_supply),
+                        status=200,
+                        mimetype='text/plain'
```
Comment (KPrasch) [inline monitor/dashboard.py:116]: Any reason why you use a different mimetype that the above?

**Review by KPrasch — APPROVED** [review]
one comment - but looks good!

**Comment by derekpierre** [comment]
@mswilkison just to confirm that you wanted these two endpoints to return the value in the same way as `http://chainz.cryptoid.info/grs/api.dws?q=totalcoins`...i.e. in plain text as opposed to `json`...

Example
```
In [2]: response = requests.get('http://chainz.cryptoid.info/grs/api.dws?q=totalcoins')

In [3]: response.headers
Out[3]: {'Date': 'Wed, 11 Nov 2020 19:47:02 GMT', 'Content-Type': 'text/plain', 'Content-Length': '17', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=d7d1eddd6e152f81605f96e0fe8097f1b1605124022; expires=Fri, 11-Dec-20 19:47:02 GMT; path=/; domain=.cryptoid.info; HttpOnly; SameSite=Lax', 'Cache-Control': 'public,s-maxage=5,max-age=5', 'ETag': 'W/"30293e294239"', 'Access-Control-Allow-Origin': '*', 'CF-Cache-Status': 'DYNAMIC', 'cf-request-id': '065a71f6fb0000252034019000000001', 'Report-To': '{"endpoints":[{"url":"https:\\/\\/a.nel.cloudflare.com\\/report?s=v2oTIP%2BQ386oNWL89RBBDP9lLNsqcpnt4Z2oqFxOs0BxArNfJDwfTCNk3moHzJ9WRGtxaDShqxXTljDPHIykR3PwckeDLzTSMRGth%2BjiI3oFKcCs9g%3D%3D"}],"group":"cf-nel","max_age":604800}', 'NEL': '{"report_to":"cf-nel","max_age":604800}', 'Server': 'cloudflare', 'CF-RAY': '5f0a85d199f52520-IAD'}

In [4]: response.headers['Content-Type']
Out[4]: 'text/plain'
```

**Comment by mswilkison** [comment]
> @mswilkison just to confirm that you wanted these two endpoints to return the value in the same way as `http://chainz.cryptoid.info/grs/api.dws?q=totalcoins`...i.e. in plain text as opposed to `json`...
> 
> Example
> 
> ```
> In [2]: response = requests.get('http://chainz.cryptoid.info/grs/api.dws?q=totalcoins')
> 
> In [3]: response.headers
> Out[3]: {'Date': 'Wed, 11 Nov 2020 19:47:02 GMT', 'Content-Type': 'text/plain', 'Content-Length': '17', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=d7d1eddd6e152f81605f96e0fe8097f1b1605124022; expires=Fri, 11-Dec-20 19:47:02 GMT; path=/; domain=.cryptoid.info; HttpOnly; SameSite=Lax', 'Cache-Control': 'public,s-maxage=5,max-age=5', 'ETag': 'W/"30293e294239"', 'Access-Control-Allow-Origin': '*', 'CF-Cache-Status': 'DYNAMIC', 'cf-request-id': '065a71f6fb0000252034019000000001', 'Report-To': '{"endpoints":[{"url":"https:\\/\\/a.nel.cloudflare.com\\/report?s=v2oTIP%2BQ386oNWL89RBBDP9lLNsqcpnt4Z2oqFxOs0BxArNfJDwfTCNk3moHzJ9WRGtxaDShqxXTljDPHIykR3PwckeDLzTSMRGth%2BjiI3oFKcCs9g%3D%3D"}],"group":"cf-nel","max_age":604800}', 'NEL': '{"report_to":"cf-nel","max_age":604800}', 'Server': 'cloudflare', 'CF-RAY': '5f0a85d199f52520-IAD'}
> 
> In [4]: response.content
> Out[4]: b'76286643.88736623'
> 
> In [5]: response.headers['Content-Type']
> Out[5]: 'text/plain'
> ```

yes thanks!

**Pull request merged**